test/test_pymbolic.py

```diff
@@ -723,7 +723,8 @@ def test_flop_counter():
     y = prim.Variable("y")
     z = prim.Variable("z")
 
-    subexpr = prim.CommonSubexpression(3 * (x**2 + y + z))
+    subexpr = prim.CommonSubexpression(3 * (x**2 + y + z),
```

Comment:
Use `make_cse` for these, to reduce grossness?

Reply:
Done in https://github.com/inducer/pymbolic/compare/9a62b35b5063d40cdbbb2084414e124141f3406b..7eba5378d83bfd5d6146ed9a94f16478758a801a.